modules/core/shared/src/main/scala/gem/enum/GnirsFpuOther.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS FPU Other.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsFpuOther(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val obsolete: Boolean
+) extends Product with Serializable
+
+object GnirsFpuOther {
+
+  /** @group Constructors */ case object Ifu extends GnirsFpuOther("Ifu", "IFU", "IFU", true)
```

Comment:
Technically speaking, the GNIRS decker slide supports two IFUs (1 and 2).  Currently neither are installed although this may change in the future.

Reply:
We also need to take into account that at some point we'll need to import previous observations, so if we already have observations with IFU enabled, I think it's worth including it even if it's no longer installed.